client/src/router/routes.js

```diff
@@ -141,6 +133,7 @@ export const Routes = [
     props: true,
     meta: { toTop: true, smoothScroll: true, requiredAuth: true },
     component: () => import("../views/UserProfile.vue"),
+    children: [],
```

Comment:
was macht das?

Reply:
Vergessen zu entfernen...

Wollte da was mit children ausprobieren, aber dafür muss die Komponente auch router-view nutzen. Das tut aber unsere Komponente nicht. Macht an der Stelle auch keinen Sinn das umzubauen.

Man könnte dann Routen halt nesten. Also ProfileEdit wäre dann als Child in Profile. Aber wiegesagt, für uns an der Stelle nicht wirklich von Interesse.